### PR TITLE
Add type annotations to main.py

### PR DESCRIPTION
**Pull request by ciplazar**
#87

### CODE REVIEW

File: apistar/main.py [hunk 1]
```diff
@@ -27,18 +28,16 @@ def get_current_app():
     if not hasattr(module, 'app'):
         raise ConfigurationError("The app.py module did not contain an 'app' variable.")
 
-    app = module.app
+    app = module.app  # type: ignore
```
Comment (ciplazar) [inline apistar/main.py]: Not sure how else to do this one since mypy complains about `module` not having an `app` attribute otherwise.

Comment (tomchristie) [inline apistar/main.py]: How about we use getattr(module, 'app', None) and do 'if app is None' instead of our 'hasattr' check above?

**Comment by tomchristie** [comment]
Great stuff thanks! Have made one suggestion that might be worth trying.

**Comment by ciplazar** [comment]
Nice one!

**Comment by tomchristie** [comment]
Ace, thanks!

**Pull request merged**